tensorrt_llm/_torch/pyexecutor/kv_cache_connector.py

```diff
@@ -570,3 +577,7 @@ def layer_pre_hook(self, module, *args):
 
     def layer_post_hook(self, module, *args):
         self.worker.save_kv_layer(module.layer_idx, torch.cuda.current_stream())
+
+    def wait_for_initialization(self):
+        if self.scheduler is not None:
+            self.scheduler.wait_for_initialization()
```

Comment:
Please run `pre-commit install` in the project root directory and resubmit to fix the formatting.